Bang.Tests/DuelSpecification.cs

```diff
@@ -0,0 +1,169 @@
+        {
+            var gamePlay = InitGame();
+            (Player actor, Player victim, BangGameCard duelCard) = ChoosePlayers(gamePlay);
+            
+            // Act
+            actor.PlayCard(duelCard, victim);
+
+            // Assert
+            gamePlay.GetTopCardFromDiscarded().Should().Be(duelCard);
+            actor.Hand.Should().NotContain(duelCard);
+        }
+
+        [Fact]
+        public void If_victim_plays_does_not_defense_in_duel_he_or_she_will_lose_one_life()
+        {
+            var gamePlay = InitGame();
+            (Player actor, Player victim, BangGameCard duelCard) = ChoosePlayers(gamePlay);
+            
+            actor.PlayCard(duelCard, victim);
+
+            var healthBefore = victim.PlayerTablet.Health;
+            // Act
+            victim.NotDefense();
+            
+            // Assert
+            victim.LifePoints.Should().Be(healthBefore - 1);
+        }
+        
+        [Fact]
+        public void If_victim_replies_with_bang_card_and_attacker_does_not_defence_attacker_will_lose_life()
+        {
+            var gamePlay = InitGame();
+            (Player actor, Player victim, BangGameCard duelCard) = ChoosePlayers(gamePlay);
+            
+            actor.PlayCard(duelCard, victim);
+            
+            var bangCard = new BangCardType().ClubsSeven();
+            victim.AddCardToHand(bangCard);
+
+            var healthBefore = actor.PlayerTablet.Health;
+            // Act
+            victim.Defense(bangCard);
+            actor.NotDefense();
+            
+            // Assert
+            actor.LifePoints.Should().Be(healthBefore - 1);
+        }
+        
+        [Fact]
+        public void If_victim_replies_with_bang_card_then_it_will_be_discarded()
+        {
+            var gamePlay = InitGame();
+            (Player actor, Player victim, BangGameCard duelCard) = ChoosePlayers(gamePlay);
+            
+            actor.PlayCard(duelCard, victim);
+            
+            var bangCard = new BangCardType().ClubsSeven();
+            victim.AddCardToHand(bangCard);
+
+            // Act
+            victim.Defense(bangCard);
+            
+            // Assert
+            gamePlay.GetTopCardFromDiscarded().Should().Be(bangCard);
+        }
+        
+        [Fact]
+        public void If_victim_replies_with_bang_card_then_victim_will_lose_bang_card()
+        {
+            var gamePlay = InitGame();
+            (Player actor, Player victim, BangGameCard duelCard) = ChoosePlayers(gamePlay);
+            
+            actor.PlayCard(duelCard, victim);
+            
+            var bangCard = new BangCardType().ClubsSeven();
+            victim.AddCardToHand(bangCard);
+
+            // Act
+            victim.Defense(bangCard);
+            
+            // Assert
+            victim.Hand.Should().NotContain(bangCard);
+        }
+
+        [Fact]
+        public void If_victim_and_attacker_play_by_one_bang_card_then_victim_will_lose_one_life()
```

Comment:
It seems the test name doesn't reflect correctly the logic. Here attacker plays 2 bang cards

Reply:
I don't see that. Attacker player plays `Duel` and `Bang`
```
actor.PlayCard(duelCard, victim);
victim.Defense(anotherBangCard);
actor.Defense(bangCard);
```